src/core/localState.js

```diff
@@ -43,14 +44,14 @@ export class LocalState {
         return data;
       })
       .catch((error) => {
-        log.error(`Error with localForage.getItem("${this.id}"): ${error}`);
+        log.info(`Error with localForage.getItem("${this.id}"): ${error}`);
```

Comment:
This component was written prior to having error boundaries. Since the errors get thrown, they will automatically go to Sentry, right? I think so. That's why I turned it down to `info`.

Reply:
I still don't understand, sorry. Kibana aggregates those log calls, it is important to respect the different levels.

Reply:
If I leave them as `error` then Sentry would get two entries for the same error. This would add a lot of noise. My thought was to only send the error to Sentry once but still log these messages for debugging purposes.